library/common/config/config.cc

```diff
@@ -36,6 +36,7 @@ const std::string config_header = R"(
 - &dns_refresh_rate 60s
 - &dns_fail_base_interval 2s
 - &dns_fail_max_interval 10s
+- &dns_query_timeout 25s
```

Comment:
Putting a default so no one is burnt by the upstream 5s default. I am working on a parallel PR to make it configurable.

Reply:
https://github.com/envoyproxy/envoy-mobile/pull/1583